usaspending_api/database_scripts/matview_generator/universal_award_matview.json

```diff
@@ -103,7 +105,9 @@
     "WHERE",
     "  \"awards\".\"latest_transaction_id\" IS NOT NULL AND",
     "  (\"awards\".\"category\" IS NOT NULL or \"contract_data\".\"pulled_from\"='IDV') AND",
-    "  latest_transaction.\"action_date\" >= '2007-10-01'"
+    "  latest_transaction.\"action_date\" >= '2007-10-01'",
+    "ORDER BY",
+    "  latest_transaction.\"action_date\" DESC"
```

Comment:
Also why are we adding `order by`s? Shouldn't this just add performance given this is just populating/updating the matviews?

Reply:
adding order by to *hopefully* structure the matview for better performance when gathering hundreds/thousands of rows. Since the time_period is a range, it helps to have those columns as close as possible on disk. The clustering was removed which helped performance-wise but took way too much time when creating the matviews